infrastructure/simulation-tool/src/operations/deposit.ts

```diff
@@ -0,0 +1,95 @@
+import { Wallet as RollupWallet, RootstockOperation } from '@rsksmart/rif-rollup-js-sdk';
+import { sleep } from '@rsksmart/rif-rollup-js-sdk/build/utils';
+import { BigNumber, ContractReceipt, Wallet as EthersWallet, constants } from 'ethers';
+import config from '../utils/config.utils';
+import { getRandomBigNumber } from '../utils/number.utils';
+import { chooseRandomWallet } from '../utils/wallet.utils';
+
+type PreparedDeposit = Omit<Parameters<RollupWallet['depositToSyncFromRootstock']>[number], 'amount'> & {
+    amount: BigNumber;
+    from: RollupWallet;
+};
+
+type DepositResult = {
+    opL1Receipt: ContractReceipt;
+};
+
+const prepareDeposit = (l2sender: RollupWallet, recipientAddress?: string, amount?: BigNumber): PreparedDeposit => {
+    const [minAmount, maxAmount] = config.weiLimits.deposit;
+
+    return {
+        amount: amount || getRandomBigNumber(minAmount, maxAmount),
+        depositTo: recipientAddress || l2sender.address(),
```

Comment:
```suggestion
        to: recipientAddress || l2sender.address(),
```

Reply:
`depositToSyncFromRootstock` takes `depositTo`, not `to`.